Dart/thirdPartySrc/analysisServer/org/dartlang/analysis/server/protocol/AvailableSuggestion.java

```diff
@@ -122,7 +122,7 @@ public boolean equals(Object obj) {
 
   public static AvailableSuggestion fromJson(JsonObject jsonObject) {
     String label = jsonObject.get("label").getAsString();
-    String declaringLibraryUri = jsonObject.get("declaringLibraryUri") != null ? jsonObject.get("declaringLibraryUri").getAsString() : null;
+    String declaringLibraryUri = jsonObject.get("declaringLibraryUri").getAsString();
```

Comment:
I'll keep this line unchanged just for the compatibility with older Dart SDK versions.